bin/rbenv_hook_install.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../" && pwd )"
+
+if [ -z $GEM_NAME ]; then
+  GEM_NAME="manpages"
+fi
+if ! $(gem list -i "$GEM_NAME") ;then
+  gem install "$GEM_NAME"
+fi
+
+if [[ -n "$(command -v git)" && \
+        "$(basename $(git config --local remote.origin.url))" == "manpages.git" ]]; then
+  DIR=$SCRIPT_DIR
+else
+  LIB_DIR="$(dirname "$(gem which manpages)")"
+  DIR="$(readlink -f  ${LIB_DIR}/../)"
+fi
+
+if [ "$DIR" != "$(rbenv root)" ]; then
+  source ${DIR}/libexec/vars.sh
+
+  mkdir -p "${PREFIX_MAN}"
+
+  mkdir -p $EXEC_HOOK_DIR
+  mkdir -p $INSTALL_HOOK_DIR
+
+  cp "${DIR}/lib/version-name-change-man.bash" $EXEC_HOOK_DIR/
+  cp "${DIR}/lib/install-man.bash" $INSTALL_HOOK_DIR/
+
+  ln -s $PREFIX_MAN "${RBENV_ROOT}/man"
+
+cat <<EOM
+  # add the following to ~/.bashrc:
+MANPATH="${RBENV_ROOT}/man:\${MANPATH}"
```

Comment:
This is not needed if you change the directory to `${RBENV_ROOT}/share/man` . That way man should automatically find it.  Can you try that?